functions/Get-DbaMsdtc.ps1

```diff
@@ -71,17 +71,21 @@ Get-DbaMsdtc -ComputerName srv0042 | Out-Gridview
 
 Get DTC status for the computer srv0042 and show in a grid view
 
+
+
 #>
 	
 	[CmdletBinding()]
 	Param (
-		[Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
-		[string[]]$Computername
+		[string[]]$Computername = $env:COMPUTERNAME
 	)
 	
-	BEGIN
+BEGIN
 	{
+    $functionName = "Get-DbaMsDtc"
+    $ComputerName = $ComputerName | ForEach-Object {$_.split("\")[0]} | Select-Object -Unique

```

Comment:
yes @FriedrichWeinmann  we had some discussion on this and decided to go for this as a general approach, putting user friendliness first. So both pulling computernames out in case a user feeds us instance names, and making computername unique before looping through.

Thanks, @wsmelton